bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -5184,6 +5169,26 @@ void ReplicaImp::finalizeExecution() {
   }
 }
 
+void ReplicaImp::updateExecutedPathMetrics(const CommitPath executionPath, uint16_t numOfRequests) {
+  metric_total_finished_consensuses_++;
+  switch (executionPath) {
+    case CommitPath::OPTIMISTIC_FAST:
+      metric_total_fastPath_++;
+      metric_total_fastPath_requests_ += numOfRequests;
+      break;
+    case CommitPath::FAST_WITH_THRESHOLD:
+      metric_fast_threshold_path_count_++;
+      break;
+    case CommitPath::SLOW:
+      metric_total_slowPath_++;
+      metric_total_slowPath_requests_ += numOfRequests;
+      break;
+    case CommitPath::NA:
+      break;
+  }
+  LOG_INFO(GL, "Commit path metric update " << KVLOG(std::to_string((int8_t)executionPath)));
```

Comment:
move to DEBUG, we have the commit path as part of the MDC

Reply:
This wasn't supposed to be committed, I'll remove it.